server/lib/fetch.py

```diff
@@ -436,10 +436,15 @@ def triples(nodes, out=True):
 
 
 def descendent_places(nodes, descendent_type):
-  return property_values(nodes,
-                         'containedInPlace+',
-                         out=False,
-                         constraints='{{typeOf:{}}}'.format(descendent_type))
+  # When the only node being requested is also the descendent_type, fetch all nodes of that type.
```

Comment:
Can you add a TODO to re-visit this. Maybe have a bug to track too.

Reply:
Done. b/302167251